### PR TITLE
Retire Topics

### PR DESCRIPTION
**Pull request by sareg0**
#### Is this solving an issue already in our backlog? If so, please mention which one
https://github.com/rubycorns/rorganize.it/issues/637

#### Special things about this PR to be considered
There are a lot of removals. I tried to be careful, but could do with a set of vigilant eyes :)

#### Do you have any open Questions you need to get answered?
did I delete all the right things. Did I miss anything?

#### Please outline the areas that should be tested, if there are any:
A good 'ol test of the whole site is wise, but making sure that the groups page still works properly is directly related to this.

### CODE REVIEW

File: app/assets/stylesheets/custom/_profiles.scss [hunk 1]
```diff
@@ -134,23 +134,6 @@
...
-}
-.topics-list .action-buttons {
-  margin: 0 auto 1rem 0;
-}
-
-.topic-comments,
-.topic-comment-form h3 {
-  margin-top: 1rem;
-}
-
```
Comment (PragTob) [inline app/assets/stylesheets/custom/_profiles.scss]: :+1: 

(lots of dev I know delete everything about a feature... but the css so good that you thought of it!)

Comment (sareg0) [reply app/assets/stylesheets/custom/_profiles.scss]: all credit to the people who kept it all organised and in one spot :)

**Comment by lislis** [comment]
sweet!
wooooop

**Comment by PragTob** [comment]
I'll take that as a yes

**Pull request merged**